src/gulpfile.babel.js

```diff
@@ -33,6 +35,9 @@ const config = require( './wpgulp.config.js' );
  */
 const gulp = require( 'gulp' ); // Gulp of-course.
 
+// Generates zip of theme.
```

Comment:
WP Gulp is used by both themes and plugins. Adding something just for themes is not right. Care to test for both and adjust the lingo?

Reply:
@naveenkharwar you mention the word `theme` it should be `theme/plugin` since WPGulp works with both.